package.json

```diff
@@ -3,19 +3,17 @@
   "version": "0.4.1",
   "description": "Get the metadata from a Google Cloud Platform environment",
   "repository": "stephenplusplus/gcp-metadata",
-  "main": "index.js",
-  "files": [
```

Comment:
Can you add a files section back? It is useful in preventing unneeded files from getting shipped in the package.

Reply:
ooooops